scripts/commands/setrank.lua

```diff
@@ -0,0 +1,37 @@
+---------------------------------------------------------------------------------------------------
+-- func: setrank
+-- desc: Sets the players rank.
+---------------------------------------------------------------------------------------------------
+
+cmdprops =
+{
+    permission = 1,
+    parameters = "si"
+};
```

Comment:
we dropped semicolons awhile back too, just fyi (they do nothing in lua)

Reply:
copy and paste left over